src/objects/zif_abapgit_oo_object_fnc.intf.abap

```diff
@@ -1,141 +1,146 @@
-      RAISING
-        zcx_abapgit_exception,
-    create_sotr
-      IMPORTING
-        iv_package TYPE devclass
-        it_sotr    TYPE zif_abapgit_definitions=>ty_sotr_tt
-      RAISING
-        zcx_abapgit_exception,
-    create_documentation
-      IMPORTING
-        it_lines       TYPE tlinetab
-        iv_object_name TYPE dokhl-object
-        iv_language    TYPE spras
-      RAISING
-        zcx_abapgit_exception,
-    get_includes
-      IMPORTING
-        iv_object_name     TYPE sobj_name
-      RETURNING
-        VALUE(rt_includes) TYPE ty_includes_tt
-      RAISING
-        zcx_abapgit_exception,
-    exists
-      IMPORTING
-        is_object_name   TYPE seoclskey
-      RETURNING
-        VALUE(rv_exists) TYPE abap_bool,
-    serialize_abap
-      IMPORTING
-        is_class_key     TYPE seoclskey
-        iv_type          TYPE seop_include_ext_app OPTIONAL
-      RETURNING
-        VALUE(rt_source) TYPE zif_abapgit_definitions=>ty_string_tt
-      RAISING
-        zcx_abapgit_exception
-        cx_sy_dyn_call_error,
-    get_skip_test_classes
-      RETURNING
-        VALUE(rv_skip) TYPE abap_bool,
-    get_class_properties
-      IMPORTING
-        is_class_key               TYPE seoclskey
-      RETURNING
-        VALUE(rs_class_properties) TYPE vseoclass
-      RAISING
-        zcx_abapgit_exception,
-    get_interface_properties
-      IMPORTING
-        is_interface_key               TYPE seoclskey
-      RETURNING
-        VALUE(rs_interface_properties) TYPE vseointerf
-      RAISING
-        zcx_abapgit_exception,
-    read_text_pool
-      IMPORTING
-        iv_class_name       TYPE seoclsname
-        iv_language         TYPE spras
-      RETURNING
-        VALUE(rt_text_pool) TYPE textpool_table,
-    read_documentation
-      IMPORTING
-        iv_class_name   TYPE seoclsname
-        iv_language     TYPE spras
-      RETURNING
-        VALUE(rt_lines) TYPE tlinetab,
-    read_sotr
-      IMPORTING
-        iv_object_name TYPE sobj_name
-      RETURNING
-        VALUE(rt_sotr) TYPE zif_abapgit_definitions=>ty_sotr_tt
-      RAISING
-        zcx_abapgit_exception,
-    read_descriptions
-      IMPORTING
-        iv_obejct_name         TYPE seoclsname
-      RETURNING
-        VALUE(rt_descriptions) TYPE zif_abapgit_definitions=>ty_seocompotx_tt,
-    delete
-      IMPORTING
-        is_deletion_key TYPE seoclskey
-      RAISING
-        zcx_abapgit_exception,
-    read_superclass
-      IMPORTING
-        iv_classname         TYPE seoclsname
-      RETURNING
-        VALUE(rv_superclass) TYPE seoclsname,
-    read_attributes
-      IMPORTING
-        iv_object_name       TYPE seoclsname
-      RETURNING
-        VALUE(rt_attributes) TYPE zif_abapgit_definitions=>ty_obj_attribute_tt.
+  TYPES:
+    BEGIN OF ty_includes,
+      programm TYPE programm,
+    END OF ty_includes .
+  TYPES:
+    ty_includes_tt TYPE STANDARD TABLE OF ty_includes WITH DEFAULT KEY .
+
+  METHODS create
```

Comment:
pretty printing, please move to separate PR

Reply:
I restore that class to do only minimum changes needed

Reply:
nice, its now down to around half the size of the original PR 😅